sw/device/lib/dif/dif_uart.c

```diff
@@ -80,6 +82,19 @@ static bool uart_irq_offset_get(dif_uart_interrupt_t irq_type,
   return true;
 }
 
+static void uart_reset(const dif_uart_t *uart) {
+  mmio_region_write32(uart->base_addr, UART_CTRL_REG_OFFSET, 0u);
```

Comment:
I don't see the point of 0u/1u as opposed to 0/1 and haven't seen it being used widely in other projects. But doesn't really matter to me much either.

Reply:
It was more to be consistent with the rest of the DIF, I don't really have much preference.

Reply:
Yeah, feel free to leave it in whatever state you feel is best.